lib/rules/media-feature-range-notation/__tests__/index.mjs

```diff
@@ -151,6 +151,43 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['context', { except: ['exact-value'] }],
+	fix: true,
+	computeEditInfo: true,
+
+	accept: [
+		{
+			code: '@media (width: 1px) {}',
+			description: 'plain media feature in prefix notation with exact value',
+		},
+		{
+			code: '@media (width = 1px) {}',
```

Comment:
The option should reject this testcase as it's an `except` rather than an `ignore` option. From our [developer guide](https://stylelint.io/developer-guide/rules#add-options):

> "ignore" skips-over a particular pattern
> "except" inverts the primary option for a particular pattern

We prefer adding `except` over `ignore` options as they allow users to enforce stricter conventions.

Let's rework the logic of the rule to do this. Here's an example of an `except` option being reserved in another rule:

https://github.com/stylelint/stylelint/blob/9a4e2b2bc5ced683dcbb67d5eaa62e2e832e489b/lib/rules/function-url-quotes/index.mjs#L165-L169

Reply:
Reworked